### PR TITLE
Avoid implicit conversion when var arg constructor take a single argument

### PR DESCRIPTION
**Pull request by Twon**

### CODE REVIEW

File: TALK.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ class dyn_optional {
   public:
     // Constructors
     dyn_optional() noexcept;
-    template <typename ...Us> dyn_optional(Us&& ...us);
+    template <typename ...Us> explict dyn_optional(Us&& ...us);
```
Comment (jbcoe) [inline TALK.md:69]: Necessary for slideware??

Comment (Twon) [reply TALK.md:69]: Quite possibly... just wanted to make sure it was not missed.  Feel free to close

Comment (jbcoe) [reply TALK.md:69]: Added to our (growing) list of exclusions.

Comment (jbcoe) [reply TALK.md:69]: Thanks for this - critical to be on top of what we have and have not done. This I just forgot.

**Pull request closed without merging**